omnilingo/main.py

```diff
@@ -0,0 +1,52 @@
+"""OmniLingo CLI Interface."""
+
+from pathlib import Path
+
+import click
+
+import omnilingo.cache as _cache
+import omnilingo.server as server
+
+
+@click.group()
+@click.option(
+    "--cache-path",
+    type=lambda x: Path(click.Path(exists=True, file_okay=False, writable=True)(x)),
+    default=_cache.DEFAULT_PATH,
+    help="Path to cache directory.",
+)
+@click.pass_context
+def main(ctx: click.Context, cache_path: Path) -> None:
+    """Omnilingo Control Commands."""
+    ctx.obj = cache_path
+
+
+@main.command()
+@click.option(
+    "--port",
+    type=click.IntRange(0, 65535),
+    default=5001,
+    help="Port for web server to listen on.",
+    envvar="FLASK_PORT",
+)
+@click.option("--host", default="127.0.0.1", help="IP Aaddress for web server to listen on.")
+@click.pass_obj
+def serve(cache_path: Path, port: int, host: str) -> None:
+    """Start Web Server."""
+    languages = [p.name for p in Path("cache/").glob("*")]
```

Comment:
Will this also glob the `.voc` files ?

Reply:
This was copied verbatim from the existing main.py but the glob is for everything in ./cache/ which I'll be anchoring explicitly in a follow up pull request.